src/library_pthread.js

```diff
@@ -639,7 +742,12 @@ var LibraryPThread = {
       worker.ref();
     }
 #endif
-    worker.postMessage(msg, threadParams.transferList);
+    worker.postMessage(msg, threadParams.transferList.concat([channel.port1]));
+    PThread.messageRelay.postMessage({
+      'cmd': 'create',
```

Comment:
Perhaps add a comment here.. Such as "Send one end of the relay channel to the newly created thread, and the other end of the messageRelay worker"?